### ISSUE TITLE
LabyrinthJS: No selection-indicator is visible while linking ideas

### ISSUE THREAD

**Issue by 44yu5h** [llaske/sugarizer#1585]
While linking ideas, no selection-indicator of selected idea was visible thereby creating confusion. 

Before:

- No selection-indicator
- Last selected idea stayed selected.

![Screencastfrom2024-04-0605-29-13-ezgif com-video-to-gif-converter](https://github.com/llaske/sugarizer/assets/130254793/65aa2bb5-0daa-4dc5-bca2-7408732154ea)

After (#1584):

- Added a selection-indicator to the currently selected idea 
- Auto-deselect after making a link.

![Screencastfrom2024-04-0605-31-08-ezgif com-video-to-gif-converter](https://github.com/llaske/sugarizer/assets/130254793/8ef27bb9-0e21-4c03-ae2b-035ce7ebb0c9)

**Comment by 44yu5h** [comment]
@llaske Agree, but how is one supposed to know which item is selected without an indicator?

**Comment by llaske** [comment]
Oh got it. You talk about the "link ideas" mode, not about the "text idea" mode.
Most of the time, links are created in "text idea" mode and the last node is always selected in this mode.
But you're right, it's not the case in "link ideas" mode.

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/1584

**Comment by llaske** [comment]
We're not closing issue until a new version is release
